django_restql/fields.py

```diff
@@ -29,15 +30,19 @@ def to_representation(self, value):
 class _ReplaceableField(object):
     pass
 
+
 class _WritableField(object):
     pass
 
-def BaseNestedFieldSerializerFactory(*args, 
-                                     accept_pk=False, 
-                                     create_ops=[ADD, CREATE], 
-                                     update_ops=[ADD, CREATE, REMOVE, UPDATE], 
-                                     serializer_class=None, 
-                                     **kwargs):
+
+def BaseNestedFieldSerializerFactory(
+    *args,
+    accept_pk=False,
+    create_ops=[ADD, CREATE],
+    update_ops=[ADD, CREATE, REMOVE, UPDATE],
+    serializer_class=None,
+    **kwargs
+):
```

Comment:
> Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest.

https://www.python.org/dev/peps/pep-0008/#id17

Reply:
Ah ok, so I didn't indent this enough times, right? I'll add another indent.